tools/scripts/update-docker-tags.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -ex
+# Retags all docker images from latest Environments build
+# tools/scripts/update-docker-tags.sh NEW_VERSION [--release]
+
+if [ "$#" -lt 1 ] || [ "$2" != "--release" ] || [ "$#" -gt 2 ]; then
+    echo "usage: $0 NEW_VERSION [--release]" >&2
+    exit 1
+fi
+
+# check for dirty changes
+if [[ -n "$(git status --porcelain)" ]]; then
+    echo "untracked or dirty files are not allowed, cleanup before running update-docker-tags.sh"
```

Comment:
Trivial change I overlooked the first time - maybe only include if you're also changing something else.
```suggestion
    echo "untracked or dirty files are not allowed, clean up before running update-docker-tags.sh"
```